frontend/src/App.vue

```diff
@@ -1,67 +1,119 @@
+<template>
+  <form @submit.prevent="submitForm">
+    <main class="wrapper">
+      <div class="container-main">
+        <PersonalInfo />
+
+        <Wrapper>
```

Comment:
we can use wrapper component inside each view, so `App.vue` is not bigger than necessary

Reply:
I wrapped all the components inside of wrapper component, but I left the Line slot untouched because I don't use it in personal info

Reply:
hmmmm, what's the point of using a wrapper component for PersonalInfo if it has no title and no line? that was the whole point of extracting the wrapper...

Reply:
also not sure what the problem was with removing usage of the wrapper component from `App.vue`